### PR TITLE
[2.13.x] DDF-4244 Decrease time to load Admin UI

### PR DESCRIPTION
**Pull request by peterhuffer**
#### What does this PR do?
Decrease amount of time it takes to load the Admin UI by using a populated map of ObjectClassDefinitions instead of fetching it per config in a loop.

#### Who is reviewing it? 
@Bdthomson @kcover @brjeter @stustison 

#### Select relevant component teams: 
@codice/security 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@pklinef 

#### How should this be tested?
Ensure the Admin UI loads within a second or 2 by inspecting the Admin UI network request in your browsers dev tools.

#### Any background context you want to provide?
Admin UI was taking 5-30 seconds to load.

#### What are the relevant tickets?
[DDF-4244](https://codice.atlassian.net/browse/DDF-4244)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by peterhuffer** [comment]
build now

File: platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java [hunk 1]
```diff
@@ -212,7 +212,7 @@ private Bundle getBoundBundle(Configuration config) {
                     + service.getId()
                     + "_disabled))");
         if (configs != null) {
-          addConfigurationData(service, configs);
+          addConfigurationData(service, configs, getPidObjectClasses(metaTypeInformationByBundle));
```
Comment (pklinef) [inline platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java]: ✏️  I know you got this patch from me but something I forgot to do was to make a local variable to hold the results from `getPidObjectClasses`.  It is now called and used 3 times in this method.

Comment (peterhuffer) [reply platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java]: Updated 👍

**Comment by peterhuffer** [comment]
aborted build to build with new commit
build now

**Comment by kcover** [comment]
Manually tested ✔️  Took less than a second.

**Pull request merged**